packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -204,10 +208,12 @@ async function handleJSONFile(err, data) {
 module.exports = async (callback: (error?: any) => number) => {
   try {
     argv = require('minimist')(process.argv.slice(5), {
-      string: ['network', 'grants', 'start_gold', 'output_file', 'really'],
+      string: ['network', 'from', 'grants', 'start_gold', 'output_file', 'really'],
     })
+    if (argv.from === '' || argv.from == null) {
```

Comment:
Can we remove this? Would prefer this need to be specified explicitly

Reply:
I'll see if I can - this is used to allow for the `devchain` commands in `yarn;reset` to still be able to call thsi script without needing to access `accounts`. I'll see if I can access it there